.pre-commit-config.yaml

```diff
@@ -97,7 +97,7 @@ repos:
     -   id: pyright_reportGeneralTypeIssues
         # note: assumes python env is setup and activated
         name: pyright reportGeneralTypeIssues
-        entry: pyright --skipunannotated -p pyright_reportGeneralTypeIssues.json
+        entry: pyright -p pyright_reportGeneralTypeIssues.json
```

Comment:
So that no new type errors are introduced in `tseries/holiday.py` (but many other files needed to be added to the ignore list for now).